src/GitHub.VisualStudio/Views/GitHubPane/PullRequestDetailView.xaml

```diff
@@ -43,6 +43,8 @@
                 <ghfvs:SharedDictionaryManager Source="pack://application:,,,/GitHub.UI;component/Assets/Markdown.xaml" />
             </ResourceDictionary.MergedDictionaries>
 
+            <v:ViewLocator x:Key="viewLocator"/>
```

Comment:
You shouldn't need a `ViewLocator` here. There's already one on the root `GitHubPaneView`.